### ISSUE TITLE
Examples fail to build

### ISSUE THREAD

**Issue by m-cat** [maidsafe/safe_client_libs#992]
I tried running the `self_authentication` example and the build fails:

```
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!   unwrap! called on Result::Err                                              !
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
safe_app/build.rs:73,9 in build_script_build::bindings

Err([Error { level: Error, span: None, message: "bindgen can not handle `#[repr(C)]` enums with non-unit variants" }])
```

I'm guessing it's due to the change that explicitly assigned values to the variants of the `Permission` enum.

**Comment by nbaksalyar** [comment]
Linked issue: #1008

**Comment by m-cat** [comment]
Closed via #1008
